### PR TITLE
PR: Debug Press Releases Listings

### PR DESCRIPTION
**Pull request by ttran-Therisa**
## Description
Relates to #[16629](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/16629). (or closes?)

Creating a draft PR to aid in debugging and obtain advice

### CODE REVIEW

**Review by nfpappas-oddball — APPROVED** [review]
Looks good to me, may still want tim or jr to look at it though

File: src/data/queries/index.ts [hunk 1]
```diff
@@ -64,6 +66,8 @@ export const QUERIES_MAP = {
   [RESOURCE_TYPES.EVENT_LISTING]: EventListing,
   [RESOURCE_TYPES.PERSON_PROFILE]: PersonProfile,
   [RESOURCE_TYPES.PRESS_RELEASE]: PressRelease,
+  [`${RESOURCE_TYPES.PRESS_RELEASE}--teaser` as const]: PressReleaseTeaser,
```
Comment (mreed101) [inline src/data/queries/index.ts:69]: Wondering if this should be an element '__teaser' instead of a modifier '--teaser'

Comment (ttran-Therisa) [reply src/data/queries/index.ts:69]: Issue was in listingPages.ts and incorrectly used the wrong pathing. Initially, placed 'press-releases' when it should have been 'news-releases'

File: playwright/tests/pressReleaseListing.spec.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import { test, expect } from '@/playwright/utils/next-test'
+
+test.describe('pressReleaseListing', () => {
+  test('pressReleaseListing page renders with stories that can be navigated to', async ({
+    page,
+  }) => {
+    await page.goto('/southern-nevada-health-care/news-releases')
```
[the remark this answers is not in the document]
Comment (ttran-Therisa) [reply playwright/tests/pressReleaseListing.spec.ts:7]: The location is arbitrary. The location is 80% influenced on what appears in the api-explorer and 20% of the content appearing on prod that I wish to simulate.

**Review by mreed101 — APPROVED** [review]

**Pull request merged**